pyproject.toml

```diff
@@ -83,6 +83,8 @@ psycopg2-binary = { version = "*", optional = true }
 pika = { version = "*", optional = true }
 redis = { version = "*", optional = true }
 selenium = { version = "*", optional = true }
+anyio = "^4.3.0"
+pre-commit = "^3.6.2"
```

Comment:
for these, 
- make `anyio` a dev dependency - you don't actually need `anyio` to run redis
- I tend not to install `pre-commit` under poetry because then you will need poetry-ception - just use `pip install pre-commit` for that one!